### PR TITLE
Update LUT FAQ for adapter upgrade issue

### PR DESCRIPTION
**Pull request by genlu**
FYI @ManishJayaswal

### CODE REVIEW

File: docs/test/live-unit-testing-faq.md [hunk 1]
```diff
@@ -51,6 +51,16 @@ Live Unit testing may not work for one of the following reasons:
     </RunSettings> 
    ``` 
 
+## Why does Live Unit Testing show incorrect coverage after upgrading referenced Visual Studio test adapter in projects to supported version?
+
+- If there is multiple projects in the solution reference test adapter NuGet package, all of them need to be upgraded to the supported version.
+
```
Comment (ManishJayaswal) [inline docs/test/live-unit-testing-faq.md]: This FAQ is currently not up-to-date with what's in the blob. @rpetrusha is going to bring it to par with FAQ and then you can update this PR. I think the blog post mentions the first part of this FAQ.

File: docs/test/live-unit-testing-faq.md [hunk 2]
```diff
@@ -51,6 +51,16 @@ Live Unit testing may not work for one of the following reasons:
     </RunSettings> 
    ``` 
 
+## Why does Live Unit Testing show incorrect coverage after upgrading referenced Visual Studio test adapter in projects to supported version?
```
Comment (rpetrusha) [inline docs/test/live-unit-testing-faq.md]: This is phrased awkwardly. Better: "Why does Live Unit Testing show incorrect coverage after you upgrade the test adapter referenced in your Visual Studio Projects to the supported version?"

File: docs/test/live-unit-testing-faq.md [hunk 3]
```diff
@@ -51,6 +51,16 @@ Live Unit testing may not work for one of the following reasons:
...
+
+- If there is multiple projects in the solution reference test adapter NuGet package, all of them need to be upgraded to the supported version.
```
Comment (rpetrusha) [inline docs/test/live-unit-testing-faq.md]: This should be preceded by **Answer:** and a paragraph break. 
Better: "If multiple projects in the solution reference the NuGet test adapter package, each of them must be upgraded to the supported version."

File: docs/test/live-unit-testing-faq.md [hunk 4]
```diff
@@ -51,6 +51,16 @@ Live Unit testing may not work for one of the following reasons:
...
+
+- Make sure the MSBuild .props file imported from test adapter package is correctly updated as well. Check the NuGet package version/path of the import, which can usually be found near the top of the project file like the following:
```
Comment (rpetrusha) [inline docs/test/live-unit-testing-faq.md]: test adapter package --> the test adapter package.
comma after "project file"

**Comment by genlu** [comment]
@rpetrusha I have updated the PR base on your comments. Thanks!

**Comment by rpetrusha** [comment]
Thanks for making the changes, @genlu!

**Pull request merged**